### ISSUE TITLE
bigtable: add keys-only argument to cbt read to view keys without large column values

### ISSUE THREAD

**Issue by jkleckner** [googleapis/google-cloud-go#1717]
When examining tables using `cbt`, it would be very handy to be able to list keys only without potentially very large cell values.

This could be done by adding an argument `keys-only` for the `read` command to suppress printing of cell values.

Thus, something like:
```
cbt --project test_project --instance test_instance \
    read \
    test_table \
    count=10 \
    keys-only=true \
    start=$'start_prefix\x1Fsub_prefix'
```

**Comment by Pega88** [comment]
+1 on this. The only way around it now is to set the column filter with a very minimal column family/column qualifier, which is still reading more data than needed. passing an empty column filter does also not give the desired result (gives no result)

**Comment by DeaconDesperado** [comment]
:+1:

**Comment by limjix** [comment]
+1 on this too.

**Comment by jimfulton** [comment]
I wonder if it would be better to be able to set a maximum output width for values, with output greater than the max width elided.

That way, you'd still see the smaller values and a sample of the larger ones.

**Comment by scordata** [comment]
+1

**Comment by telpirion** [comment]
This feature has been implemented in `cbt` version 0.12.0. (The `cbt` tool has also moved to another [repo](https://github.com/googleapis/cloud-bigtable-cbt-cli).)
